agent.py

```
@@ -78,17 +94,20 @@ class RequestAnalyseVoiceCall(BaseModel):
 
 
 class Kai(Agent):
-    def __init__(self) -> None:
+    def __init__(self, *args, **kwargs) -> None:
         prompt = langfuse.get_prompt("kai_voice_call_prompt")
-        super().__init__(
-            instructions=prompt.compile(),
-        )
+        if not 'instructions' in kwargs:
```

Ruff (0.12.2)

Test for membership should be `not in`

Convert to `not in`

(E713)